config/locales/fr.yml

```diff
@@ -290,8 +290,11 @@ fr:
       header: Dépendances de %{title}
     gem_members:
       authors_header: Auteurs
+      not_using_mfa_warning_show:
```

Comment:
Maybe we should research and add notice text for other languages?

Reply:
I'm open to contributions here. I'd rather not translate it directly using something like Google Translate